presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/TestArrowFlightIntegrationSmokeTest.java

```diff
@@ -65,8 +65,8 @@ protected QueryRunner createQueryRunner()
     public void close()
             throws InterruptedException
     {
+        arrowFlightQueryRunner.close();
         server.close();
         allocator.close();
-        arrowFlightQueryRunner.close();
```

Comment:
This is needed to ensure the correct order of closing

Reply:
although it's unrelated to native testing. I'll revert this too and I'll open another PR for some of these small fixes.